diff --git a/submodules/helix/helix-term/Cargo.toml b/submodules/helix/helix-term/Cargo.toml
index bb4f114d..b0515e32 100644
--- a/submodules/helix/helix-term/Cargo.toml
+++ b/submodules/helix/helix-term/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "helix-term"
-description = "A post-modern text editor."
+description = "Steel-enabled fork of the post-modern text editor."
 include = ["src/**/*", "README.md"]
 default-run = "hx"
 version.workspace = true
@@ -29,9 +29,12 @@ assets = [
   { source = "../contrib/Helix.desktop", dest = "/usr/share/applications/Helix.desktop", mode = "644" },
   { source = "../contrib/helix.png", dest = "/usr/share/icons/hicolor/256x256/apps/helix.png", mode = "644" },
 ]
+provides = "editor"
+conflicts = "hx"
+suggests = ["steel-forge", "steel-interpreter", "steel-language-server"]
 
 [features]
-default = ["git"] # Add steel here for development
+default = ["git", "steel"] # Add steel here for development
 unicode-lines = ["helix-core/unicode-lines", "helix-view/unicode-lines"]
 integration = ["helix-event/integration_test"]
 git = ["helix-vcs/git"]
